src/connectors/hits-per-page/connectHitsPerPage.js

```diff
@@ -197,6 +200,38 @@ The first one will be picked, you should probably set only one default value`
       dispose() {
         unmountFn();
       },
+
+      getWidgetState(fullState, { state }) {
```

Comment:
According to the change in `getWidgetSearchParameters`:

- `fullState` should be renamed `uiState`
- `state` should be renamed `searchParam`